### ISSUE TITLE
Plugin prints empty lines

### ISSUE THREAD

**Issue by thasmo** [olegskl/gulp-stylelint#22]
When using `console: true`, empty lines are printed on the console when no errors were found:
```
[13:53:35] Starting 'styles:lint'...
[13:53:36]



[13:53:36] Finished 'styles:lint' after 795 ms
```
That's my config:
```
{
	syntax: 'scss',
	reporters: [
		{
			formatter: 'string',
			console: true
		}
	]
}
```
Not sure if it's a bug or I didn't configure it correctly but the current behavior is weird.

**Comment by olegskl** [comment]
Thanks for reporting this. Will be fixed soon.

**Comment by olegskl** [comment]
This issue has been fixed and released in 2.0.2, so I'm closing it. Feel free to reopen if necessary.
